changelog.d/545.misc

```diff
@@ -0,0 +1 @@
+Don't block invites containing `invite_client_location` when using "http" in the keyword block list.
```

Comment:
Can we replace mentions of `invite_client_location` with `web_client_location` in the PR title, changelog and comments?

Sydent calls it the latter and has no mention of the former.

Reply:
Sure thing I think I got them all.